src/DotLiquid/Hash.cs

```diff
@@ -165,10 +152,14 @@ public Hash()
 
         #endregion
 
-        public void Merge(IDictionary<string, object> otherValues)
+        /// <summary>
+        /// Merge the contents of the given Dicionary into this Hash.
+        /// <summary>
+        public Hash Merge(IDictionary<string, object> otherValues)

```

Comment:
This is a breaking change on a public method. There is no reason for this change other than skimping on two lines.

Reply:
What does a change of return type from void to Hash break?
My intention is to enable fluent use of Merge, for example:
`var merged = Hash.FromDictionary(dictionary1).Merge(dictionary2);`

Reply:
If the developer rebuilds the code it should be fine. But if there is just an assembly redirect to a new version, I am not sure the method resolution at runtime will still work

Reply:
I requested change because (a) I'm not sure like @daviburg says and (b) there was no reason for it - nothing in our code is fluent style (c) if we are chaining a constructor with a merge - that just means we are missing a constructor with a dictionary (See https://github.com/dotliquid/dotliquid/pull/441/files#diff-fc52e9cceed37df79819e1e24e3657a1e091af0d2a293047135da7a2b991f4f9R163 where I did just that). FYI - DataSet Merge returns void as do pretty much every method in .NET with the name merge.

Reply:
I've reverted the Merge changes. Also FWIW, I don't think these changes will resolve #442.

Is there a guide anywhere I can check to avoid future style conflicts?

Reply:
I don't think the dotLiquid project has yet a documented code style. But that would be a good thing to start now that we have several concurrently active contributors (yah!).